src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -132,7 +132,7 @@ var render = function render(interaction) {
 
             hideTooltip($input);
 
-            if ($input.val().length && regex.test($input.val())) {
+            if (($input.val().length && regex.test($input.val())) || ($input.val().length && regex.test($input.val()))) {
```

Comment:
That doesn't make any sense, does it?
The second condition just repeats the first one as-is.

Reply:
Some days, I don't know what I do :D